js/index.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function(){
+	console.log('i see you');
+	/*set up a listener on the #city-type id
+		if then to capture all possible inputs
+		use $.val to store user input
+		pass the value to a variable
+		use .uppercase and save to a new variable
+		add appropriate class to the body
+		pray to a higher being
+		amen
+	*/
+	$('#submit-btn').click(function(event){
+		event.preventDefault(); 
+		console.log('oh do you?');
+		var cityName = $('#city-type').val();
+		var cityUpper = cityName.toUpperCase();
+
+		if (cityUpper==='NYC'|| cityUpper==='NEW YORK CITY' || cityUpper==='NEW YORK') {
+			$('body').addClass('nyc');
```

Comment:
You got the right idea here. There's only one problem:

If the use types 'SF' and then we add the class `'sf'` to the `body` tag - everything is fine. But then the user types in `'new york'` and then we add the `'nyc'` class to the body. The problem is, we didn't remove any of the other city class names that may have been added previously.

What you should do before adding the appropriate class based on what the user typed, you should first do some clean up on the `body` tag. This could look something like this:

```js
// Remove all the cities 
$('body').removeClass('nyc la sf austin sydney');

if (cityUpper === 'NYC' || cityUpper === 'NEW YORK' || cityUpper === 'NEW YORK CITY') {
   // now that we've removed any classes, we can now ADD the one we want
   $('body').addClass('nyc');
} // ...
```

The problem with not remove the the class names is you'll end up with something like this as the user types in different cities:

1. User types in "New York"
```html
<body class="nyc"></body> <!-- so far so good -->
```
2. User types in "SF"
```html
<body class="nyc  sf"></body> <!-- Uh oh! Now we have two classes -->
```
3. User types in "LA"
```html
<body class="nyc  sf  la"></body> <!-- And now three ...  -->
```

What we want is this:
1. User types in "New York"
```html
<body class="nyc"></body> <!-- Just nyc -->
```
2. User types in "SF"
```html
<body class="sf"></body> <!-- Just sf -->
```
3. User types in "LA"
```html
<body class="la"></body> <!-- And just la  -->
```